### PR TITLE
Rooms images

### LINKED ISSUE #12: Rooms should have many images
- [ ] Use active storage
  - [ ] Add have many images for room
  - [ ] Add input images

### PR DESCRIPTION
**Pull request by condef5**
closed #12

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -59,3 +59,5 @@ gem 'rspec-rails'
 gem 'devise'
 gem 'omniauth-facebook'
 gem 'omniauth-github'
+
+gem 'mini_magick', '~> 4.8'
```
Comment (liannivin) [inline Gemfile]: Could you explain what mini_magick does, please?

File: app/views/admin/hotels/_form.html.erb [hunk 2]
```diff
@@ -25,6 +25,16 @@
...
+  <div class="field">
+    <%= f.label :image %><br />
+    <%= f.file_field :image %>
+  </div>
+
+  <div class="field">
+    <%= f.label :gallery %><br />
+    <%= f.file_field :gallery, :multiple => true%>
+  </div>
+
```
Comment (liannivin) [inline app/views/admin/hotels/_form.html.erb]: nice!

File: app/views/admin/rooms/show.html.erb [hunk 3]
```diff
@@ -5,7 +5,20 @@
...
+  <% end %>
+  <% if @room.gallery.attached? %>
+    <p>Gallery:</p>
+    <div class="gallery">
+      <% @room.gallery.each do |image| %>
+        <%= image_tag image.variant(resize: "300x300") %>
+      <% end %>
+    </div>
+  <% end %>
 </div>
```
Comment (liannivin) [inline app/views/admin/rooms/show.html.erb]: Nice implementation. Now every image will resize to a standar size.

File: Gemfile (hunk 1 above)
Comment (condef5) [reply Gemfile]: Sure, the mini_magick allows us to resize images super easily.
It is used like this:
```
<%= image_tag image.variant(resize: "300x300") %>
```
You can read more [here](https://github.com/minimagick/minimagick)

**Pull request merged**